slave-ansible/Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM openshift/jenkins-slave-base-centos7
+FROM docker.io/fhwendy/jenkins-slave-base-centos7:RHMAP15237
 
 MAINTAINER Oleg Matskiv <omatskiv@redhat.com>
```

Comment:
Does this entry cause an extra layer to be created in the resulting image? Is this necessary?

(I realise this isn't the point of this PR, just noticed it on a screen share)